.github/actions/release/action.yml

```diff
@@ -1,9 +1,15 @@
 name: "tag version"
 description: "Tag package"
 inputs:
-  jira_token:
+  jira_api_token:
     description: "Token to authenticate to Jira"
     required: true
+  jira_user_email:
+    description: "Email linked to token"
+    required: true
+  jira_project_id:
```

Comment:
you need to modify the line 53 to use it
`,\"projectId\":11789,` -> `,\"projectId\":${{ inputs.jira_project_id }},`

Reply:
should indeed be using GHA input and not raw value